### ISSUE TITLE
EventException on TNT in non-EHM world

### ISSUE THREAD

**Issue by SlimeDog** [MLG-Fortress/ExtraHardMode#148]
Spigot 1.13.2 latest
ExtraHardMode 3.13.0-3.3.2-411

This error occurred 15 times when I exploded TNT in another world (ie., not the EHM-configured world), probably once for each sand block blown upwards (which then have to fall back to earth). It is not clear why EMH would trap an action that occurred in another world.

```
17:10:31 ERROR]: Could not pass event EntityChangeBlockEvent to ExtraHardMode v3.13.0-3.3.2-411
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:308) ~[spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callEntityChangeBlockEvent(CraftEventFactory.java:798) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callEntityChangeBlockEvent(CraftEventFactory.java:790) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.EntityFallingBlock.tick(EntityFallingBlock.java:128) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.World.entityJoinedWorld(World.java:1380) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.World.g(World.java:1347) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.World.tickEntities(World.java:1181) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.WorldServer.tickEntities(WorldServer.java:601) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:972) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_181]
Caused by: java.lang.NoSuchFieldError: Legacy field ordinal: LEGACY_SAND
        at org.bukkit.craftbukkit.v1_13_R2.util.CraftLegacy.modern_ordinal(CraftLegacy.java:254) ~[spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        at com.extrahardmode.module.BlockModule.getDroppedMaterial(BlockModule.java:477) ~[?:?]
        at com.extrahardmode.features.Explosions.handleLandedBlocksFromPhysics(Explosions.java:290) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor350.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_181]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_181]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:304) ~[spigot-1.13.2.jar-2018-11-24-2111:git-Spigot-a1f2566-97b0057]
        ... 15 more
```

**Comment by RoboMWM** [comment]
Closed in 60b5c0cdd3540bcb94e52d61977428bdb57c57f9

**Comment by SlimeDog** [comment]
Confirmed issue fixed in EHM 3.13.0-60b5c0c. I did not verify that all EHM features are intact.
